Bottom: 43f4cee964f45b3bb43dab4a7064731aae526e2e
Top:    2a8b0f85189b5c7ec299abef507e2c8ade047677
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2015-01-04 17:03:49 +0000

borast: Win32: Comment #define in borast-compiler-private.h


---

diff --git a/src/borast/borast-compiler-private.h b/src/borast/borast-compiler-private.h
index f886f40..165b42e 100644
--- a/src/borast/borast-compiler-private.h
+++ b/src/borast/borast-compiler-private.h
@@ -160,7 +160,7 @@
 #endif
 
 #if (defined(__WIN32__) && !defined(__WINE__)) || defined(_MSC_VER)
-#define snprintf _snprintf
+//#define snprintf _snprintf
 #define popen _popen
 #define pclose _pclose
 #define hypot _hypot
